actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -471,36 +471,25 @@ def mail_to(email_address, name = nil, html_options = {}, &block)
       # True if the current request URI was generated by the given +options+.
       #
       # ==== Examples
-      # Let's say we're in the <tt>http://www.example.com/shop/checkout?order=desc</tt> action.
+      # Let's say we're in the <tt>http://www.example.com/shop/checkout?order=desc&page=1</tt> action.
       #
       #   current_page?(action: 'process')
       #   # => false
       #
-      #   current_page?(controller: 'shop', action: 'checkout')
-      #   # => true
-      #
-      #   current_page?(controller: 'shop', action: 'checkout', order: 'asc')
-      #   # => false
-      #
       #   current_page?(action: 'checkout')
       #   # => true
       #
       #   current_page?(controller: 'library', action: 'checkout')
       #   # => false
       #
-      #   current_page?('http://www.example.com/shop/checkout')
-      #   # => true
-      #
-      #   current_page?('/shop/checkout')
+      #   current_page?(controller: 'shop', action: 'checkout')
       #   # => true
       #
-      # Let's say we're in the <tt>http://www.example.com/shop/checkout?order=desc&page=1</tt> action.
-      #
-      #   current_page?(action: 'process')
+      #   current_page?(controller: 'shop', action: 'checkout', order: 'asc')
       #   # => false
       #
-      #   current_page?(controller: 'shop', action: 'checkout')
-      #   # => true
+      #   current_page?(controller: 'shop', action: 'checkout', order: 'desc')
```

Comment:
Generally the order has two values i.e. `asc` or `desc` in most of the cases, so I thought it would be good to show both examples. Let's remove it if you insist.